<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognitionSystem</class>
 <widget class="QDialog" name="RecognitionSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>414</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RecognitionSystem</string>
  </property>
  <property name="windowOpacity">
   <double>0.990000000000000</double>
  </property>
  <widget class="QGroupBox" name="ImageBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>360</width>
     <height>295</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>295</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>295</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>345</width>
      <height>285</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>345</width>
      <height>285</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>345</width>
      <height>285</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="plate7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="operateBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="operateImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="plateLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>94</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>License plate No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.761364, y1:0.08, x2:1, y2:1, stop:0 rgba(0, 78, 152, 255), stop:1 rgba(136, 255, 255, 255));
border-left-style:solid;border-left-width:1px;border-left-color:black;
border-right-style:solid;border-right-width:1px;border-right-color:black;
border-top-style:solid;border-top-width:1px;border-top-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Traffic monitoring system 2012</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 86, 131);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;
border-right-style:solid;border-right-width:1px;border-right-color:black;
border-top-style:solid;border-top-width:1px;border-top-color:black;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout2">
    <item>
     <widget class="QPushButton" name="locateButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Locate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cutButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Cut</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="matchButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Match</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="allInOneButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>All in one</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="plateNumOutput">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>281</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout3">
    <item>
     <widget class="QPushButton" name="Binaryzation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Binaryzation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="corrupt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Corrupt&amp;Expand</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="meanfilter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>MeanFilter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>ImageBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>operateBox</zorder>
  <zorder>plateLabel</zorder>
  <zorder>bgLabel</zorder>
  <zorder>bgLabel_2</zorder>
  <zorder>ExitButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>plateNumOutput</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>RecognitionSystem</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
